src/helper-modules/ordnance-survey/ordnance-survey.module.ts

```diff
@@ -0,0 +1,26 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { KEY as ORDNANCE_SURVEY_CONFIG_KEY, OrdnanceSurveyConfig } from '@ukef/config/ordnance-survey.config';
+import { HttpModule } from '@ukef/modules/http/http.module';
+
+import { OrdnanceSurveyService } from './ordnance-survey.service';
+
+@Module({
+  imports: [
+    HttpModule.registerAsync({
+      imports: [ConfigModule],
+      inject: [ConfigService],
+      useFactory: (configService: ConfigService) => {
+        const { baseUrl, maxRedirects, timeout } = configService.get<OrdnanceSurveyConfig>(ORDNANCE_SURVEY_CONFIG_KEY);
```

Comment:
Just a small touch `baseUrl: baseURL` so `return {...` can be all consistent wth refering to the same variable names.

Reply:
Changed